test/e2e/scenarios/add_cluster.go

```diff
@@ -0,0 +1,15 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("Adding exisitng network policy to newly added cluster", func() {
+	PIt("Should implement exisitng network policy in newly added cluster", func() {
+		By("Create a new cluster")
+		By("Discover the newly created cluster")
+		By("creating pod1 with lable1 in cluster1 in namespace1")
+		By("creating pod3 with lable3 in cluster3 in namespace3")
```

Comment:
should the labels be same? Also, what is the scope of the NetworkPolicy.

Reply:
Labels need to be different. Network policy can be any. The point here is to show that whatever the policy is, it is implemented in the newly created cluster.